Mathlib/RingTheory/DedekindDomain/Different.lean

```diff
@@ -310,3 +339,84 @@ lemma dual_le_dual :
   rwa [dual_ne_zero_iff]
 
 end FractionalIdeal
+
+variable (B)
+variable [IsDedekindDomain B]
+
+/-- The different ideal of an extension of integral domains `B/A` is the inverse of the dual of `A`
+as an ideal of `B`. See `coeIdeal_differentIdeal` and `coeSubmodule_differentIdeal`. -/
+def differentIdeal [NoZeroSMulDivisors A B] : Ideal B :=
+  (1 / Submodule.traceDual A (FractionRing A) 1 : Submodule B (FractionRing B)).comap
```

Comment:
Looking at Mathlib.RingTheory.DedekindDomain.Ideal I have the impression we use `⁻¹` rather than `1 /`, you can maybe do the same (but feel to keep `1 / ` if you have a good reason to do so).

Reply:
I did this so that we don't need an `[IsDedekindDomain B]` instance considering your comment on the last PR.